astropy/coordinates/transformations.py

```diff
@@ -519,6 +519,7 @@ def to_dot_graph(self, priorities=True, addnodes=[], savefn=None,
 
         # generate simple dot format graph
         lines = ['digraph AstropyCoordinateTransformGraph {']
+        lines.append('graph [rankdir=LR]')
```

Comment:
can we make this direction a kwarg? that way sunpy can keep the current behaviour while astropy also gets the fix?

Reply:
Of course this could be made a kwarg, but SunPy ultimately wants the left->right direction as well.  It's so much more readable given the shape of the nodes.  I just need to add a quick hack in SunPy so that, in SunPy's graph, the HCRS node is the same rank as the ICRS node, and then everything looks great.

Reply:
OK, that case, given the approvals above, I go ahead and merge this.